src/pages/index.astro

```diff
@@ -63,6 +66,89 @@ const t = useTranslations(lang);
       </div>
     </div>
   </section>
+  <section class="section audienceSection">
+    <div class="contentWrapper audience-grid">
+      <div class="audience-card publisher">
+        <div>
+          <h2>I am a publisher</h2>
+          <h3 class="font-h6">I want to earn from my content</h3>
+          <PillTag pills={data.pillTags.publisher} />
+        </div>
+        <div class="media-cta">
+          <img
+            width="460"
+            height="460"
+            class="inactive audience-img"
+            aria-hidden="true"
+            src="/img/hero-publisher-inactive.png"
+            loading="lazy"
+          />
+          <img
+            width="460"
+            height="460"
+            class="active audience-img"
+            aria-hidden="true"
+            src="/img/hero-publisher.png"
+          />
+          <a class="btn primary flexible" href="/publisher" data-umami-event="Home page link - Publishers">Get started as a publisher</a>
```

Comment:
Right after click (and when focus styles are active), the button text/bg are both black:
<img width="1003" height="95" alt="image" src="https://github.com/user-attachments/assets/fdeaf400-6d70-4ae4-b586-2d8f22dc3495" />

Reply:
Fixed, thanks